addons/resource.language.en_gb/resources/strings.po

```diff
@@ -17814,7 +17820,29 @@ msgctxt "#36614"
 msgid "Show add-ons that are currently running in the background."
 msgstr ""
 
-#empty strings from id 36615 to 36899
+#: system/settings/settings.xml
+msgctxt "#36615"
+msgid "Unknown sources"
+msgstr ""
+
+#. Description of setting "System -> Add-ons -> Unknown sources" with label #36615
+#: system/settings/settings.xml
+msgctxt "#36616"
+msgid "Allow installation of add-ons from unknown sources."
+msgstr ""
+
+#: xbmc/addons/GUIWindowAddonBrowser.cpp
```

Comment:
please also add a contextual comment here. Like if it's used as headline of a dialog, if it's a warning message for something, etc. Translators don't check the code, so they should get an idea on how stuff is used in order to create correct translations. It also helps other devs in case the want to reuse labels.